lib/public/components/common/navigation/breadcrumbs.js

```diff
@@ -15,8 +15,8 @@
 import { h, iconChevronRight } from '/js/src/index.js';
 
 /**
- * Create Breadcrumbs out of provided items, each items is preceded by rightChevron Icon
+ * Create Breadcrumbs out of provided items which are separated with rightChevron Icon
  * @param {Component[]} items items in breadcrumbs
  * @return {Component} breadcrumbs
  */
-export const breadcrumbs = (...items) => h('.flex-row.g1.items-center', items.flatMap((item) => [iconChevronRight(), item]));
+export const breadcrumbs = (items) => h('.flex-row.g1.items-center', items.flatMap((item) => [iconChevronRight(), item]).slice(1));
```

Comment:
This solution is indeed more elegant than mine. Could you please also update the other places where we do the other way aound? In this PR is fine, there is not much and this has no impact.

Reply:
applied